### DISCUSSION TITLE
GPU Instances for CodeSpaces

### DISCUSSION THREAD

**Discussion by aosama** [community/community#108108] [Codespaces]
### Select Topic Area

Product Feedback

### Body

First my complements on a very good product, I would love to be able to launch a codespaces instance with GPU to power Model training. Codespaces is convenient to use coupled with Jupyter within vscode its a perfect combination for training ML models. hope this can go into consideration.

**Comment by man-chi** [comment]
@github team

me too, may I know how to apply for a GPU instance?
